### ISSUE TITLE
Insufficient color contrast ratio on elements

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/tcf#7]
- [ ] Giving, Impact, and Stories
  - Giving (expected 4.5:1, found 3.2:1)
  - Impact (expected 4.5:1, found 2.08:1)
  - Stories (expected 4.5:1, found 2.61:1)
- [ ] Donate button in site header. (expected 4.5:1, found 3.2:1)
- [ ] Newsletter text in site footer (expected 4.5:1, found 2.61:1)
